<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonDossierFrame</class>
 <widget class="QFrame" name="MonDossierFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QPushButton" name="btn_valider">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>550</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_annuler">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>550</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>551</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Acte de Naissance : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="champ_acte_naissance"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_acte_naissance">
      <property name="text">
       <string>Parcourir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>390</y>
     <width>551</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Photo :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="champ_photo"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_photo">
      <property name="text">
       <string>Parcourir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>440</y>
     <width>551</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Diplôme : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="champ_diplome"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_diplome">
      <property name="text">
       <string>Parcourir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>280</y>
     <width>551</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>CV :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="champ_cv"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_cv">
      <property name="text">
       <string>Parcourir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>461</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Prénom : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="prenom_candidat"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>461</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nom : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nom_candidat"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
